balboa-http/src/main/scala/com/socrata/balboa/server/EntityJSON.scala

```diff
@@ -0,0 +1,5 @@
+package com.socrata.balboa.server
+
+case class EntityJSON(timestamp: Long, metrics: Map[String, MetricJSON])
```

Comment:
It would be better to do this by updating the existing objects instead of creating new ones, but as discussed offline, this will be consolidated in a following review.